lib/TWCManager/Control/HTTPControl.py

```diff
@@ -431,6 +424,54 @@ def do_API_POST(self):
             self.end_headers()
             self.wfile.write("".encode("utf-8"))
 
+        elif self.url.path == "/api/setScheduledChargingSettings":
```

Comment:
I'm considering the relationship between this and the UI. On the one hand I am prototyping it using POST form handling which means this API call could happily coexist without impacting that handling, but I am also considering what it might look like long term. There's so many form elements involved that I wonder if we'll ever have a single API call that would control all of the scheduling via a single call, or whether we'd just change certain parameters as in here. We could keep them distinct for now, with the UI using a form and POST variables allowing this to be merged in parallel, and then I'll see if it is feasible to unify the settings under this API endpoint, but it may involve making breaking changes to the API spec, that's really the key risk